packages/tooltip/src/index.js

```diff
@@ -268,6 +268,21 @@ export function useTooltip({
 
   useEffect(() => checkStyles("tooltip"));
 
```

Comment:
Could simplify this a bit:

```
useEffect(() => {
    const listener = event => {
      if ((event.key === "Escape" || event.key === "Esc") && state === VISIBLE) {
        transition("selectWithKeyboard")
      }
    };
    document.addEventListener("keydown", listener);
    return () => document.removeEventListener("keydown", listener);
  }, []);
```

Reply:
Yeah but then my very important, totally helpful `console.log("whoa")` wouldn't work!